lib/api_metric.go

```diff
@@ -0,0 +1 @@
+package sebak
```

Comment:
Unused ?

Reply:
Another PR will add code for this file.

Reply:
Then let's not add this empty file here, but in another PR